prow/cmd/crier/main.go

```diff
@@ -124,6 +126,7 @@ func (o *options) parseArgs(fs *flag.FlagSet, args []string) error {
 	fs.Float64Var(&o.k8sReportFraction, "kubernetes-report-fraction", 1.0, "Approximate portion of jobs to report pod information for, if kubernetes-blob-storage-workers are enabled (0 - > none, 1.0 -> all)")
 	fs.StringVar(&o.slackTokenFile, "slack-token-file", "", "Path to a Slack token file")
 	fs.StringVar(&o.reportAgent, "report-agent", "", "Only report specified agent - empty means report to all agents (effective for github and Slack only)")
+	fs.IntVar(&o.resultstoreWorkers, "resultstore-workers", 0, "Number of resultstore report workers (0 means disabled)")
```

Comment:
s/resultstore report workers/ResultStore reporters/

Reply:
The "report workers" is verbatim of the other flags. Shall I change them all to "reporters" instead?

Reply:
Leaving "report workers" but renaming ResultStore with caps.